cumulusci/core/config/project_config.py

```diff
@@ -123,9 +124,13 @@ def _validate_config(self):
         self._validate_package_api_format()
 
     def _validate_package_api_format(self):
+        api_version_regex = r"^\d\d+\.0$"
```

Comment:
This is probably unnecessary optimization, but I always tend to define regexes as a constant at module scope so they only get compiled once:

    import re
    API_VERSION_RE = re.compile(r"^\d\d+\.0$")

(and then use it like `API_VERSION_RE.match(target_string)`)